sdk/servicebus/azure-messaging-servicebus-stress/stress-test-resources.bicep

```diff
@@ -1,15 +1,17 @@
 @description('The base resource name.')
-param baseName string = resourceGroup().name
+param baseResourceName string = resourceGroup().name
 
 @description('The location of the resources. By default, this is the same as the resource group.')
 param location string = resourceGroup().location
 
-var subBaseName = '${substring(baseName, 0, min(length(baseName), 12))}-${substring(guid(baseName), 0, 4)}'
+var subBaseName = '${substring(baseResourceName, 0, min(length(baseResourceName), 12))}-${substring(guid(baseResourceName), 0, 4)}'
 var serviceBusNamespaceName = '${subBaseName}-sbns'
```

Comment:
Do we need to abbreviate these? Since the namespace name is for a service bus entity, we don't need to specify it here and in the variable name.  I feel like the less exceptions we need to add to cspell, the easier it will be for newer devs to understand our code.

`var serviceBusNamespaceName = '${subBaseName}-java-stress'` Or something.

Reply:
My thoughts also apply to ehns and ehrl.  They're unique within their own Event Hub, so the rule name can be anything like "StressAuthRule" or something.

Reply:
Good suggestion! I need add some prefix in case the name may conflict, but yes, I can change to some other words to pass the cspell.